setup.cfg

```
@@ -28,7 +28,7 @@ install_requires =
     jsonlines>=3.1.0
     sqlalchemy>=2.0  # note this will be incompatible with InvenioRDM (see invenio-db)
     sqlalchemy-utils[encrypted]>=0.38.3
-    psycopg2>=2.9.6
+    psycopg>=2.9.6
```

```suggestion
    psycopg>=3.1.9
```

minor: just to make it super-clear we're in with psycopg v3